app/views/shared/_insufficient_balance_resource.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= t('dashboard.balance_warning_prefix_html', units_balance: balance.units_balance, unit: @unit) %>
   <strong><%= balance.project_type %> <%= balance.balance_object %></strong>
   <small class="pull-right hidden-xs hidden-sm">
-    <%= t('dashboard.balance_reload_message_html', last_update: balance.last_update) %>
+    <%= t('dashboard.balance_reload_message_html', last_update: local_time(balance.last_update)) %>
```

Comment:
This could just be:

```
    <%= t('dashboard.balance_reload_message_html', last_update: local_time(balance.updated_at)) %>
```

and `Balance#last_update` removed as `Balance#last_update` purpose is to make a local time human readable form of `Balance#updated_at`, which is what `local_time(balance.updated_at)` also achieves.